src/main/resources/view/PersonListCard.fxml

```diff
@@ -2,36 +2,37 @@
 
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Label?>
-<?import javafx.scene.layout.ColumnConstraints?>
 <?import javafx.scene.layout.FlowPane?>
-<?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
-
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
-  <GridPane HBox.hgrow="ALWAYS">
-    <columnConstraints>
-      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
-    </columnConstraints>
-    <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+<?import javafx.scene.control.Separator?>
+<?import javafx.scene.image.ImageView?>
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
+      prefHeight="160" maxWidth="120" alignment="CENTER">
+  <VBox alignment="CENTER" minHeight="30">
+    <VBox alignment="CENTER_LEFT" VBox.vgrow="ALWAYS" prefHeight="80">
+      <VBox VBox.vgrow="ALWAYS">
+        <Label fx:id="id" styleClass="cell_big_label" text="\$id" wrapText="true"/>
+        <VBox VBox.vgrow="ALWAYS" alignment="CENTER">
+          <Label fx:id="name" textAlignment="CENTER" text="\$first" styleClass="cell_big_label" wrapText="true"/>
+        </VBox>
+      </VBox>
+    </VBox>
+    <Separator styleClass="separator-card" maxWidth="80">
```

Comment:
Bug mentioned might have been caused by this seperator.

Reply:
Okay, fixed